types/redis/v1/redis.yaml

```diff
@@ -0,0 +1,7 @@
+imports:
```

Comment:
This is fixing #35.  Do we have to add a yaml file?

The docs say "A version is just a folder that contains a type definition. As you can see from the examples above, a type definition is just a Python or Jinja file plus an optional schema."  From that description, I assumed that jinja + jinja.schema should be enough.